src/neo/SmartContract/ApplicationEngine.Contract.cs

```diff
@@ -94,29 +94,43 @@ protected internal UInt160 CreateMultisigAccount(int m, ECPoint[] pubKeys)
             return Contract.CreateMultiSigRedeemScript(m, pubKeys).ToScriptHash();
         }
 
-        protected internal void NativeOnPersist()
+        protected internal async void NativeOnPersist()
```

Comment:
`async void` is not recommended, it's better to use `async Task`
https://docs.microsoft.com/en-us/archive/msdn-magazine/2013/march/async-await-best-practices-in-asynchronous-programming

Reply:
We need `void` here.